templates/partners/become-a-partner.html

```diff
@@ -1,508 +1,619 @@
 {% extends 'base.html' %}
 
 {% block content %}
-  <section class="p-strip--image is-dark">
-    <div class="p-strip--suru-top">
-      <div class="row">
-        <div class="col-12">
-          <h1>Become a partner</h1>
-          <p class="u-sv3">Partnering with Ubuntu and Canonical consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+<section class="p-strip--image is-dark">
+  <div class="p-strip--suru-top">
+    <div class="row">
+      <div class="col-12">
+        <h1>Become a partner</h1>
+        <p class="u-sv3">Ubuntu is an open source computing platform that is transforming the way millions of people and businesses use the client, the server and the cloud. Canonical is the company that supports it, providing strategic guidance and driving its adoption.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-4">
+        <div class="u-sv3">
+          <img src="https://assets.ubuntu.com/v1/5c9e96f7-group-38.svg" alt="">
         </div>
+        <div class="p-heading-highlight--bottom">
+          <h3 class="p-heading-highlight--bottom__item">Grow your business</h3>
```

Comment:
Couple of heading heirarchy issues:
![Screenshot 2019-10-17 at 16 39 12](https://user-images.githubusercontent.com/2376968/67026567-f3419900-f0ff-11e9-89c3-addc727959b4.png)

Perhaps "Grow your business", "Certification of products" and "Save time and money" could be p tags with `.p-heading--three`?